CustomShips.cpp

```diff
@@ -472,6 +472,27 @@ HOOK_METHOD(ShipManager, OnLoop, () -> void)
             ex->erosion.anim->OnUpdate();
         }
     }
+    
+    bool hackSoundLoop = false;
+    if (hackingSystem)
+    {
+        hackSoundLoop = hackingSystem->SoundLoop();
+    }
+    if (!hackSoundLoop)
+    {
+        ShipManager* otherShip = G_->GetShipManager((iShipId + 1)%2);
+        if (otherShip) for (auto system : otherShip->vSystemList) if (system->bUnderAttack && system->iHackEffect == 2)
+        {
+            hackSoundLoop = true;
+            break;
+        }
+        if (!hackSoundLoop) for (auto system : vSystemList) if (system->bUnderAttack && system->iHackEffect == 2)
```

Comment:
it's legal, but man I hate this, combining single-line if's and for's with a multi line brace and then all shoved on a single line with no tabbing over on multiple lines...

maybe at least space it on separate lines? I know we don't have any linting rules but... I dunno, this bugs me and probably only me.

```c++
if(!hackSoundLoop)
    for(blah)
        if(bleh) {
        }
```

I'll still approve even without any changes to this, it's just a bugs me thing.

Reply:
Fixed (pain)